client/src/components/WeatherNav.vue

```diff
@@ -80,6 +80,8 @@
           let request = new XMLHttpRequest();
           let url = 'https://weather-api.lco.global/query?site=' + site_code + '&datumname=' + 'Weather Ok To Open' +
             '&agg=False' + '&start=' + start.format() + '&end=' + end.format();
+         // for debugging only, use localhost
```

Comment:
why have this comment at all? If you want to make it easy to switch between hosts, then have a variable for host and make a comment to change that to localhost. I see queries are made from other javascript files as well, so it would be best to set a host variable in one place at the top level and share the value with any vue components that need to make queries.

Reply:
@jnation3406 I have no idea why I even put this comment here, this doesn't apply at all. You would almost never use a different hostname to query the "Ok to open" status, and I definitely didn't need to change it while on this branch because none of the work I even did touched this feature at all